workers/generic-worker/host/host.go

```diff
@@ -12,14 +13,53 @@ import (
 
 // Run is equivalent to exec.Command(name, arg...).Run() but with logging.
 func Run(name string, arg ...string) (err error) {
-	_, err = RunCommand(exec.Command(name, arg...))
+	_, err = runCommand(exec.Command(name, arg...))
 	return
 }
 
 // CombinedOutput is equivalent to exec.Command(name, arg...).CombinedOutput()
 // but with logging.
 func CombinedOutput(name string, arg ...string) (combinedOutput string, err error) {
-	return RunCommand(exec.Command(name, arg...))
+	return runCommand(exec.Command(name, arg...))
+}
+
+// Output is like CombinedOutput but only returns Standard Error output. If an
+// error is encountered, both standard error and standard output are logged.
+func Output(name string, arg ...string) (string, error) {
+	log.Printf("Running command: %s %s", shell.Escape(name), shell.Escape(arg...))
+
+	cmd := exec.Command(name, arg...)
+
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return "", err
+	}
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		return "", err
+	}
+
+	if err := cmd.Start(); err != nil {
+		return "", err
+	}
+
+	stdoutBytes, err := io.ReadAll(stdout)
+	if err != nil {
+		return "", err
+	}
+	stderrBytes, err := io.ReadAll(stderr)
+	if err != nil {
+		return "", err
+	}
+
+	if err := cmd.Wait(); err != nil {
+		log.Printf("Error running command: %v", err)
+		log.Printf("Standard output:\n%s", stdoutBytes)
+		log.Printf("Standard error:\n%s", stderrBytes)
+		return "", err
+	}
+
+	return string(stdoutBytes), nil
```

Comment:
This doesn't seem to match the docstring?

```go
// Output is like CombinedOutput but only returns Standard Error output.
```

suggests this should be:
```suggestion
	return string(stderrBytes), nil
```

Or is the docstring wrong? Or am I reading this incorrectly? 😄

Reply:
I think, based on how you're using this function, that the docstring is wrong. I've made a suggestion to fix.

Reply:
Well spotted! I've applied your fix to the docstring. Thanks!